internal/privatemessaging/groupmanager.go

```diff
@@ -204,7 +204,7 @@ func (gm *groupManager) getGroupNodes(ctx context.Context, groupHash *fftypes.By
 //
 // Errors are only returned for database issues. For validation issues, a nil group is returned without an error.
 func (gm *groupManager) ResolveInitGroup(ctx context.Context, msg *fftypes.Message) (*fftypes.Group, error) {
-	if msg.Header.Tag == string(fftypes.SystemTagDefineGroup) {
+	if msg.Header.Tag == fftypes.SystemTagDefineGroup {
```

Comment:
I got rid of the special enum type for the system tags - didn't seem to add anything, apart from a bunch of this kind of casting all over the place. They are just string constants now.